### PR TITLE
Dockerizing

### PR DESCRIPTION
**Pull request by bessarabov**
I want to store the environment in the git repo.
This simplyfies the process of recreating the README.md file.

Actually when I did PR https://github.com/cblp/yaml-sucks/pull/1 I just manually changed the README.md because I don't have pyhon3 and haskell installed on my notebook.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+RUN apt-get update
+RUN apt-get install -y \
+    libyaml-perl \
+    python3 \
+    python3-yaml \
+    software-properties-common
+
+RUN add-apt-repository "deb http://archive.ubuntu.com/ubuntu $(lsb_release -sc) universe"
+RUN apt-get update
+RUN apt-get install -y haskell-platform
```
Comment (cblp) [inline Dockerfile]: Actually, you only need `ghc` and `libghc-yaml-dev`.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+RUN cabal update
+RUN cabal install yaml
```
Comment (cblp) [inline Dockerfile]: Remove cabal invocation please.

**Comment by bessarabov** [comment]
Not sure if you have recived notifications that I have updated this PR. I've created this comment just in case github does not send mails about this actions.

**Comment by cblp** [comment]
Yes, GitHub doesn't notify about commits. Explicit commenting is a good practice.

**Pull request merged**